grails-app/views/contacts/_contactBulletins.gsp

```diff
@@ -11,6 +11,7 @@
         </thead>
         <tbody>
         <g:each in="${bulletins}" var="bulletin">
+            <g:if test="${!bulletin.getResent()}">
```

Comment:
Lazy maaan!!! :p
This logic should be in the API and is a flag for filter not for this if.